src/futures-client.ts

```diff
@@ -316,6 +316,25 @@ export class FuturesClient extends BaseRestClient {
     });
   }
 
+  /** Get All historic positions, only supports Query within 3 monthso  */
+  getHistoryPositions(
+    startTime: string,
+    endTime: string,
+    productType?: FuturesProductType,
+    symbol?: string,
+    pageSize?: number,
+    lastEndId?: string,
+  ): Promise<APIResponse<FuturesPosition[]>> {
+    return this.getPrivate('/api/mix/v1/position/history-position', {
+      startTime,
+      endTime,
+      productType,
+      symbol,
+      pageSize,
+      lastEndId,
+    });
+  }
```

Comment:
This looks great! Tiny thing, since the last 4 params are optional and you might get situations where you'd want to provide a symbol without providing a productType, for example. This is easier to manage if the fn takes an object instead of separate parameters, just like getAccountBill does below. Something like this:
```typescript
  getHistoryPositions(
    params: FuturesHistoryPositionsParams
  ): Promise<APIResponse<FuturesPosition[]>> {
   return this.getPrivate('/api/mix/v1/position/history-position', params);
  }
```

You can then create a new FuturesHistoryPositionsParams interface with these request params here:
https://github.com/tiagosiebler/bitget-api/blob/master/src/types/request/futures.ts#L38-L39